locales/it/era.yml

```diff
@@ -1,38 +1,37 @@
 ---
 it:
-  language: language
-  interviewers: interview team
-  cameramen: camera team
-  date_of_interview: date of interview
-  alias_name: alias name
+  language: lingua
+  interviewers: intervistatori
+  cameramen: riprese
+  date_of_interview: data dell'intervista
+  alias_name: pseudonimo
   begin_of_engagement:
   end_of_engagement:
-  resistance_groups: resistance groups
-  kind_of_resistance: kind of resistance
-  firstname: firstname
-  lastname: lastname
-  gender: gender
-  born_on: birthdate
-  died_on: died on
+  resistance_groups: gruppi di resistenza
+  kind_of_resistance: tipo di resistenza
+  firstname: nome
+  lastname: cognome
+  gender: genere
+  born_on: data di nascita
+  died_on: data di morte
   born_in: born in %{date}
```

Comment:
I'm in favor of `data di nascita: %{date}` but currently we show only the year (because unfortunately we don't have the exact dates of all witnesses). 

As an example: 
  GIACOMO NOTARI (BORN IN 1927)
taken from http://www.resistance-archive.org/en/testimonies/giacomo-notari/

Would `data di nascita: %{date}` work in this case, or would it be something like `anno di nascita: %{date}` or `annata %{data}`?